java/examples/build.gradle

```diff
@@ -10,15 +11,13 @@ repositories {
 }
 
 dependencies {
-    implementation 'software.amazon.glide:glide:255.255.255'
+    implementation group: 'software.amazon.glide', name: 'glide-for-redis', version: '0.4.3', classifier: osdetector.classifier
```

Comment:
why are we hard coding the version here? 
I guess this is just an example...

Reply:
TIL to make the example work and get the binaries from the maven repo.

Long answer:
The examples are also compiled in the regular build, so this needs to be working on everyone's machine. To avoid using placeholders, and to the developer/end-user go there and change for the specifics. The example is ready to use for everyone and no changes are needed it